packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3074,9 +3074,14 @@ export class ContainerRuntime
 							error,
 						);
 					} else {
+						// Default retry delay is 1 second. This can be overridden via config so that we can adjust it
+						// based on telemetry while we decide on a stable number.
+						const retryDelayMs =
+							this.mc.config.getNumber("Fluid.Summarizer.PendingOpsRetryDelayMs") ??
+							1000;
 						const error = new RetriableSummaryError(
 							"PendingMessagesInSummary",
-							1 /* retryAfterSeconds */,
+							retryDelayMs / 1000,
```

Comment:
Out of curiosity, why don't we just have everything in `Ms`?

Reply:
Good question. I had the same one :-).
That's because this can also come from ODSP errors where its `retryAfterSeconds` - https://github.com/microsoft/FluidFramework/blob/1cff49a5de59be8d17e57e5f968d14d9a82a22b1/packages/common/driver-definitions/src/driverError.ts#L152

Reply:
To keep the code in `SummaryGenerator` and `RunningSummarizer` same for both kinds of errors.